.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+env:
+  global:
+    - secure: "o1qm2dEuLetpgQaC1hCp84t0R44eUhfJRXcy/CRQunThL8kbtI8LpIqdR8nRk6c/WT6NcLoVi0TPDBYqCHvJ4MQubbIgMdUp11UeB2Z7Cx57Cey8btet4cwCtDgNM8Zt10rTSKr/WR6UDqpRaF7R/k7LhZSA2JMQX2euAsC7aotrjNOoiTzDH/Bok2LvaRHAtfU2v+udftHpzEX7WzVtYJ9k1cqFZFPlo5C/v+9MhOkx1FBOdQ5onmFUIQxz6hfSczl+mUPYcs1T4SrQtEYagoILdLySAuhTFCmPpCHiPwJaLLjLU5nG9otFLXZrrN0kI2cRr+tubW2bC3PG03H4xHP65lHwrLPNcqUX9V1Ic5dZwnMgle9oTOk7n6Sc3JZn21TK+IiHxaK8XUtUxj+hDsiby1kr8sgcGjTp259icXyifY0TmVzaALbg5NVI9rOxq3q9+LjR5070HtaDc3ZSUwO12eSWaSmV3m01VRwvR5sZdPjsZvUOWSpG2S4cto2tU0aBUCiHrS3IlmeoLlH9yTS0jcpviti4jpYqNL7YCXmxHgl6vUo4GJA+NNE1eLJqU9jzEb0ySuEI/jp6/7NQ2IAJySH1OPsux6WG3AX2uSsJ9c15kcvhHgI1NNVwkoHuR1U4CR34VFtayd3bSV0G6r6p0cbV+PUzKI9l4+cvznY="
```

Comment:
This Environment variable is the required GITHUB_TOKEN which is needed to push to GitHub Releases.

The current value is the one I was testing with my account. You need to replace this with the one you generated.